python/quadratic-exp.cpp

```diff
@@ -15,12 +15,13 @@ namespace python {
 
 namespace bp = boost::python;
 
-void exposeActivationModelQuadExp() {
-  bp::register_ptr_to_python<std::shared_ptr<ActivationModelQuadExp> >();
+template <int N>
+void exposeActivationModelQuadExpTpl(const char* name) {
```

Comment:
I would rename it like this
```suggestion
void exposeActivationModelExpTpl(const char* name) {
```

Reply:
Done